### PR TITLE
[SUPPORT]: Update Feature flag to handle threshold spam filter directly from FF

### PR DESCRIPTION
**Pull request by mcayuelas-ledger**
### 📝 Description

Addition of a parameter to the simpleHahs FF so that the threshold can be managed manually and adapted according to user feedback.

https://github.com/LedgerHQ/ledger-live/assets/112866305/ab553d0b-1159-4e96-bbc7-b4105ea0c9f3

Url sent : `
https://simplehash.api.live.ledger.com/api/v0/nfts/owners_v2?chains=polygon&wallet_addresses=0x31BD1afBf1775A1FA55Ccb34B42AC482aA51fF6E&limit=50&filters=spam_score__lte=40`

### ❓ Context

- **JIRA or GitHub link**: [LIVE-11243] 

### ✅ Checklist

Pull Requests must pass the CI and be code reviewed. Set as Draft if the PR is not ready.

- [ ] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

---

### 🧐 Checklist for the PR Reviewers

- [ ] **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- [ ] **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- [ ] **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- [ ] **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- [ ] **Any new dependencies** have been justified and documented.
- [ ] **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-11243]: https://ledgerhq.atlassian.net/browse/LIVE-11243?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by mcayuelas-ledger** [comment]
` threshold: isThresholdValid(thresold) ? Number(thresold) : 75,` I choose to do something like that because we'll implement new feature on LLM and LLD to let user the choice of this thresold

File: libs/live-nft/src/api/simplehash.ts [hunk 1]
```diff
@@ -42,24 +41,28 @@ type NftFetchOpts = {
...
   filters?: string;
+  /**
+   * spam filtering threshold, defaults to a constant %
+   */
+  threshold?: number;
 };
 const defaultOpts = {
   limit: PAGE_SIZE,
-  filters: `spam_score__lte%3D${SPAM_FILTER_THRESHOLD}`,
+  filters: `spam_score__lte%3D`,
```
Comment (KVNLS) [inline libs/live-nft/src/api/simplehash.ts:51]: I think that we can use https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams wdyt?

Comment (gre) [reply libs/live-nft/src/api/simplehash.ts:51]: btw, that default seems invalid @mcayuelas-ledger ? it doesn't have a value, it's like `spam_score__lte=`

**Review by KVNLS — APPROVED** [review]
lgtm!

**Pull request merged**